torch_xla/stablehlo.py

```diff
@@ -212,6 +214,22 @@ class StableHLOModelBundle:
   stablehlo_funcs: List[StableHLOFunc]
 
 
+# Ops that will be mapped to stablehlo.custom_call in the exported StableHLO graph.
+custom_ops_allowed_in_graph: Set[Callable] = set()
```

Comment:
Instead of a global, put this into the Config object that exported_program_to_stablehlo receives.

Also let's do per library granularity for now instead of per op.

user registers 'my_custom_library' entirely

Reply:
Done, put it in the config object.